packages/typescript-typings/types/solc/index.d.ts

```diff
@@ -87,62 +88,7 @@ declare module 'solc' {
         };
         settings: CompilerSettings;
     }
-    export type ErrorType =
-        | 'JSONError'
-        | 'IOError'
-        | 'ParserError'
-        | 'DocstringParsingError'
-        | 'SyntaxError'
-        | 'DeclarationError'
-        | 'TypeError'
-        | 'UnimplementedFeatureError'
-        | 'InternalCompilerError'
-        | 'Exception'
-        | 'CompilerError'
-        | 'FatalError'
-        | 'Warning';
-    export type ErrorSeverity = 'error' | 'warning';
-    export interface Error {
-        sourceLocation?: {
-            file: string;
-            start: number;
-            end: number;
-        };
-        type: ErrorType;
-        component: 'general' | 'ewasm';
-        severity: ErrorSeverity;
-        message: string;
-        formattedMessage?: string;
-    }
-    import { ContractAbi } from 'ethereum-types';
-    export interface StandardContractOutput {
-        abi: ContractAbi;
-        evm: {
-            bytecode: {
-                object: string;
-                sourceMap: string;
-            };
-            deployedBytecode: {
-                object: string;
-                sourceMap: string;
-            };
-        };
-    }
-    export interface StandardOutput {
-        errors: Error[];
-        sources: {
-            [fileName: string]: {
-                id: number;
-                ast?: object;
-                legacyAST?: object;
-            };
-        };
-        contracts: {
-            [fileName: string]: {
-                [contractName: string]: StandardContractOutput;
-            };
-        };
-    }
+    export { ErrorType, ErrorSeverity, SolcError, StandardContractOutput, StandardOutput } from 'ethereum-types';
```

Comment:
Let's only have one export from `ethereum-types` at the top of the file, and exported everything we need from it.